prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,10 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgress'];
+console.log(languages);
+const firstElement = languages[0];
+console.log(firstElement);
+const thirdElement = languages[2];
+console.log(thirdElement);
+const length = languages.length;
+console.log(length);
+const lastElement = length - 1;
```

Comment:
here you want to get the last element in the array by targeting the array at the index which is the length of the array minus one. `postgres` is the last element value here.

Reply:
@Zhiwutian Is this correct?
<img width="516" alt="Screenshot 2023-03-04 at 10 11 50 AM" src="https://user-images.githubusercontent.com/121325991/222922147-921e6aa0-499a-4395-88d5-ada9bd25d325.png">

<img width="552" alt="Screenshot 2023-03-04 at 10 11 58 AM" src="https://user-images.githubusercontent.com/121325991/222922151-15ba34dd-ffe3-44f4-8934-13272306c734.png">